widget/Startup.swift

```diff
@@ -1,4 +1,4 @@
-//  Copyright © 2020 Onegini. All rights reserved.
+//  Copyright © 2020 OneWelcome. All rights reserved.
 
 import OneginiSDKiOS
```

Comment:
how about the warning I mentioned, that the sdk cannot be used in the widget?

Reply:
I checked locally and I think it is on SDK side and this tick should do the trick 
![](https://user-images.githubusercontent.com/93587538/183455969-12087b4a-db4c-4d04-9ea3-5277db078741.png)